_posts/2019-01-03-dropping-puppet4.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Dropping Puppet 4 support
+date: 2019-01-03
+github_username: bastelfreak
+twitter_username: BastelsBlog
+---
+
+Puppet 4 is end of life since 2018-12-31. There isn't an official
```

Comment:
`Puppet 4 reached End of Life on 2018-12-31.` Sounds a bit better in English.

Reply:
How about

> Me remembers it like it was yesterday. In December 31 of 2018, when Puppet 4 walked the plank.

or

> Well shave me belly with a rusty razor! Puppet 4 is end of life since 2018-12-31

?